distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Aggregate_Column_Aggregator.enso

```diff
@@ -0,0 +1,149 @@
+from Standard.Base import all
+
+from Standard.Table.Data.Column as Column_Module import Column
+import Standard.Table.Data.Group_By_Key
+from Standard.Table.Data.Aggregate_Column import all
+
+## Type used to compute an aggregate column
+type Aggregate_Column_Aggregator
+    type Aggregate_Column_Aggregator initial accumulator finalizer
+
+## Given a Table and a Column create an aggregator
+new : Table->Aggregate_Column->Aggregate_Column_Aggregator
+new table column =
+    create_closure c function:(Column->Any->Integer->Any) = function (column.resolve_column table c)
```

Comment:
Hmmmmm, I don't like this name - it doesn't really tell what it does.

More importantly - this function is not necessary anymore, at least once we also merge the `resolve_columns`.

Maybe you could borrow the `resolve_columns` [from my PR](https://github.com/enso-org/enso/pull/3353/files#diff-e0cbd6c7bf1df4a90b5d0d6a147784cf422e13fc7527fb7e8407b38622f15d25R218-R267) to have it here already. Because I think we both agreed that the pattern to use here is to first do `aggregate_column.resolve_columns` and then all the other helper functions can assume that they only get these already-resolved aggregates - and thus the `create_closure` function would be obsolete because it would boil down to `function c`.